scripts/local_ipxe_server.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+
+import argparse
+import logging
+import os
+from http.server import HTTPServer, CGIHTTPRequestHandler
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--dir', type=str, help='Directory location of script relative to assisted-test-infra base directory', required=False, default=f"{os.getcwd()}/build/local_ipxe_server")
+parser.add_argument('--ip', type=str, help='Server IP', required=False, default="192.168.122.1")
+parser.add_argument('--port', type=int, help='Port to host this server with', required=False, default=8500)
+args = parser.parse_args()
+
+# Make sure the server is hosting the iPXE scripts directory
+dirpath=args.dir
+os.chdir(dirpath)
+
+# Create server object
+server_object = HTTPServer(server_address=(args.ip, args.port), RequestHandlerClass=CGIHTTPRequestHandler)
+# Start the web server
+server_object.serve_forever()
```

Comment:
Create an ipxe controller, make it run inside container (the same as we did on the proxy controller).
Scripts is more for external test preparation, e.g. create-environment, deploy-assisted-service and more

Reply:
Created an ipxe controller! How does it look?

Reply:
This is the proxy controller https://github.com/openshift/assisted-test-infra/blob/master/src/assisted_test_infra/test_infra/controllers/proxy_controller/proxy_controller.py
And this the fixture that create the controller https://github.com/openshift/assisted-test-infra/blob/master/src/tests/base_test.py#L665-#L679

Reply:
Awesome! Thank you for the pointers :D